tests/snapshots/local/component-model/a.wast/0.print

```diff
@@ -21,5 +21,5 @@
     )
   )
   (import "three" (instance (;0;) (type 1)))
-  (export "one" (value $v))
+  (export (;1;) "one" (value $v))
```

Comment:
Perhaps we should be printing `(export "one" (value $v (;1;)))`?

To me it's easier to line up the index comment with the index space when it's near the sort, like we do for definitions and imports (i.e. it's not `(import (;x;) ...)`).

Reply:
Now that I say that, I guess that could be confused with saying that `$v` has index `1` when it does not.

Fine with keeping as-is, although I think this format does make it harder to grep for "where is item X with index Y declared".

Reply:
In favor of just leaving this be as the index comment is being printed where the identifier (if it had one) would be parsed in `wat` anyway.

I'll get used to it.

Reply:
Nah I definitely agree that `(export $name ...` isn't as nice as all the other textual forms. I'm hoping we can shave that yak at a later date, however, and figure out a better bikeshed.